docs/tutorial/cvmfs/what-is-cvmfs.md

```diff
@@ -35,6 +35,8 @@ support this, including:
 * [compression of data](#features-compression);
 * [verification of data integrity](#features-data-integrity);
 
+<span style="color: red;">data vs files? would keep it consistent</span>
```

Comment:
I've opened an issue to follow-up on it (see #455), so we can go ahead and merge, after discussing it with @HereThereBeDragons

```suggestion
```